DFC.ServiceTaxonomy.TestSuite/Features/Editor/SharedContent.feature

```diff
@@ -0,0 +1,60 @@
+﻿@webtest
+Feature: SharedContent
+
+Background:
+    Given I set the content type to be "SharedContent" 
+	Given I logon to the editor
+
+@Editor
+Scenario: Content for "Get help using this service" is available
+
+	Then I can navigate to the content item "Contact us" in Orchard Core core
+    Given I capture the generated URI
+	Then the values displayed in the editor match the following values and types
+    | Name          | Type | Value      |
+    | Title         | Base | Contact us |
+    | CanonicalName | Text | contact-us |
```

Comment:
`CanonicalName` is no longer a field on the `SharedContent` content type.
Instead it uses the guid from the end of the uri field.
FYI, an example event grid event currently looks like this...
```
{
  "id": "4cwg7c3bfwrf50tsq53rm59exe",
  "subject": "/content/sharedcontent/92d5e335-3668-45fa-b22c-a6037489d561",
  "data": {
    "api": "http://nationalcareers.service.gov.uk/sharedcontent/92d5e335-3668-45fa-b22c-a6037489d561",
    "versionId": "4q140sze68qs935pyxh9s1ymx7",
    "displayText": "hhh",
    "author": "admin"
  },
  "eventType": "published",
  "eventTime": "2020-05-12T08:09:29.2417205Z",
  "dataVersion": "1.0",
  "metadataVersion": "1",
  "topic": "/subscriptions/41c52171-ab77-4a1a-bc1d-cda18524be3f/resourceGroups/Stax/providers/Microsoft.EventGrid/topics/Stax"
}
```

Reply:
I've removed references to canonical name